internal/cmd/bundle/bundle.go

```diff
@@ -246,9 +256,13 @@ func (c *bundleCmd) buildAllVariants() ([]map[string]*build.Result, error) {
 	var allVariantBuildResults []map[string]*build.Result
 	for _, variant := range configureVariants {
 		builder, err := cmake.NewBuilder(&cmake.BuilderOptions{
-			ProjectDir:      c.config.ProjectDir,
-			Engine:          variant.Engine,
-			Sanitizers:      variant.Sanitizers,
+			ProjectDir: c.config.ProjectDir,
+			Engine:     variant.Engine,
+			Sanitizers: variant.Sanitizers,
+			Parallel: cmake.ParallelOptions{
+				Enabled: c.Flags().Lookup("build-jobs").Changed,
```

Comment:
`c.Flags().Lookup("build-jobs").Changed` is only true if the flag was set, not if the `build-jobs` setting was set in the `cifuzz.yaml`. I think we want this instead:

```suggestion
				Enabled: viper.IsSet("build-jobs"),
```

(same for the other usages of `c.Flags().Lookup("build-jobs").Changed`)